packages/formatting/src/core/generator.ts

```
@@ -0,0 +1,347 @@
+/**
+ * Configuration file generator
+ */
+
+import { stringify as stringifyYaml } from 'yaml';
+import type { PresetConfig, FormatterType, GeneratedConfig, FormatterDetectionResult, SetupOptions, FormatterDetection } from '../types/index.js';
+import type { Result } from '@outfitter/contracts';
+import { success, failure, makeError } from '@outfitter/contracts';
+import { GeneratedConfigSchema, FormatterTypeSchema } from '../schemas/index.js';
```

ESLint

[error] 'GeneratedConfigSchema' is defined but never used.

(@typescript-eslint/no-unused-vars)